[sda-cli] command exits with panic when just the `-config CONFIGFILE` is provided

**Describe the bug**

When the command is run with 
```sh
sda-cli -config <CONFIGFILE> 
```
It exits with panic and showing errors like
```
panic: runtime error: index out of range [1] with length 1

goroutine 1 [running]:
main.ParseArgs()
	/data3/work/BILS/project/project-sda/sda-cli/main.go:137 +0x5d8
main.main()
	/data3/work/BILS/project/project-sda/sda-cli/main.go:73 +0x38
```

**Steps to reproduce**

**Expected behavior**
- [ ] <!-- Issue specific criteria -->
It should show more user friendly message such as 
`Error: sub command is missing`. 

**Additional context**

**Estimation of size**: small

**Estimation of priority**: low

--- a/main.go
+++ b/main.go
@@ -134,6 +134,11 @@ func ParseArgs() (string, []string, string) {
 
 	// Extract the command from the 1st argument, then remove it
 	// from list of arguments.
+	if len(os.Args) < 2 {
+		fmt.Fprintf(os.Stderr, "Error: no command given\n")
+		os.Exit(1)
+
+	}
 	command := os.Args[1]
 	os.Args = append(os.Args[:1], os.Args[2:]...)
 

--- a/main.go
+++ b/main.go
@@ -137,7 +137,6 @@ func ParseArgs() (string, []string, string) {
 	if len(os.Args) < 2 {
 		fmt.Fprintf(os.Stderr, "Error: no command given\n")
 		os.Exit(1)
-
 	}
 	command := os.Args[1]
 	os.Args = append(os.Args[:1], os.Args[2:]...)

